src/domain/help/pages/termsOfUsePage/termsOfUsePage.module.scss

```diff
@@ -0,0 +1,8 @@
+.termOfUse {
+  list-style-type: none;
+}
+.termOfUse li:before {
```

Comment:
<img width="569" alt="Screenshot 2023-08-25 at 14 31 21" src="https://github.com/City-of-Helsinki/linkedcomponents-ui/assets/66477579/6fa0630d-f5dd-4a94-8d08-fc167ef6abb0">

Looks like the list items could use `var(--color-black-70);`. There is a bit of color mismatch with the body text 🤔

Reply:
Updated correct colour